dpbench/benchmarks/l2_distance/l2_distance_numba_dpex_p.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 Intel Corp.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+import numba as nb
+
+__njit = nb.njit(parallel=True, fastmath=True)
+
+
+@__njit
```

Comment:
Lets just inline nb.njit in the decorator.

Reply:
Cleaned up. Passed the tests. Pushed again.